wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -1045,19 +1064,7 @@ private void handleBlockchainStateNotification(BlockchainState blockchainState)
     }
 
     private int percentageSync() {
-        int chainHeadHeight = blockChain.getChainHead().getHeight();
-        int mostCommonChainHeight;
-        if (peerGroup == null) {
-            return 0;
-        }
-        if (peerGroup.getMostCommonChainHeight() > 0) {
-            mostCommonChainHeight = peerGroup.getMostCommonChainHeight();
-        } else {
-            mostCommonChainHeight = chainHeadHeight;
-        }
-        float percentage = ((float) chainHeadHeight / (float) mostCommonChainHeight) * 100;
-        log.info("mostCommonChainHeight: " + mostCommonChainHeight + "\tchainHeadHeight: " + chainHeadHeight + "\t" + percentage + "%\t" + config.getBestChainHeightEver());
-        return (int) percentage;
+        return syncPercentage;
```

Comment:
This is a calculation that give the percentage in terms an absolute percentage of how much the entire blockchain is synced (from block 0).  This is some left over code that should have been caught in a previous review 4 months ago when the method was changed to match what `dashj `provides in the `progress `method above.